src/main/java/frc/robot/subsystems/ArmPivot.java

```diff
@@ -3,45 +3,99 @@
 import static edu.wpi.first.units.Units.Volts;
 
 import com.ctre.phoenix6.configs.CurrentLimitsConfigs;
+import com.ctre.phoenix6.configs.MotionMagicConfigs;
+import com.ctre.phoenix6.configs.Slot0Configs;
 import com.ctre.phoenix6.configs.TalonFXConfiguration;
 import com.ctre.phoenix6.configs.TalonFXConfigurator;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
 import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.GravityTypeValue;
 import com.ctre.phoenix6.signals.NeutralModeValue;
 import edu.wpi.first.units.measure.Voltage;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Hardware;
 import java.util.function.Supplier;
 
 public class ArmPivot extends SubsystemBase {
   // Presets
-  public static final int PRESET_L1 = 0;
-  public static final int PRESET_L2_L3 = 35;
-  public static final int PRESET_L4 = 90;
-  public static final int HARDSTOP_HIGH = 90;
-  public static final int HARDSTOP_LOW = 0;
+  private final double ARMPIVOT_KP = 0.1;
+  private final double ARMPIVOT_KI = 0;
+  private final double ARMPIVOT_KD = 0;
+  private final double ARMPIVOT_KS = 0;
+  private final double ARMPIVOT_KV = 0;
+  private final double ARMPIVOT_KG = 0;
+  private final double ARMPIVOT_KA = 0;
+  public static final double PRESET_L1 = 0.0; // This is at position perpendicular to elevator
+  public static final double PRESET_L2_L3 = 45.0;
+  public static final double PRESET_L4 = 0.0;
+  public static final double PRESET_UP = 90.0; // Pointing Directly upwards
+  public static final double PRESET_DOWN = -90.0;
+  public static final double HARDSTOP_HIGH = 95.0;
+  public static final double HARDSTOP_LOW = -95.0;
+  public static final double POS_TOLERANCE = 1.0;
   public static final double PLACEHOLDER_CORAL_WEIGHT_KG = 0.8;
-  public static final double PLACEHOLDER_ALGAE_WEIGHT_KG = 1.5;
+  private static final double ARM_RATIO = (12.0 / 60.0) * (20.0 / 60.0) * (18.0 / 48.0);
```

Comment:
Please add a comment for maintainers on what this math is doing and what each number represents, to make it easier to change it in the future if needed.